### DISCUSSION TITLE
[Feature Request] Add a Image/Logo to the dashboard

### DISCUSSION THREAD

**Discussion by Surgical71** [daledavies/jump#76] [Feature Requests]
Hi there.

Is it possible to just add a feature to add a Logo/Image to the dashboard as shown below for example:
![image](https://user-images.githubusercontent.com/80483431/215729991-f44f0b74-8851-4ee2-b237-789262f27790.png)

Thanx

**Comment by daledavies** [comment]
Thanks for reaching out. It is not currently possible to do this but I'll consider it as a possible future enhancement.

I'm transferring this to a discussion.
